dialects/linux/dsock.c

```diff
@@ -75,7 +73,7 @@ static char copyright[] =
  * to the same hash backet. This makes seaching the counter part of
  * an end point easier. See get_netpeeri(). */
 #define TCPUDP_IPC_HASH(tp) ((int)(((((tp)->faddr			\
-				      + (tp)->laddr,			\
+				      + (tp)->laddr			\
 				      + (tp)->fport			\
 				      + (tp)->lport			\
 				      + (tp)->proto) * 31415) >> 3)	\
```

Comment:
Should use distinct prime factors (large, distinct) to minimize number of collisions. Currently this hash function has obvious collisions that can occur easily.

Reply:
Thank you.
However, in this pull request, I would like to focus on suppressing the warnings reported by the I compiler.
So, could you make a pull request after I merged this pull request?

Reply:
Can try to prepare one.